ci/build.py

```diff
@@ -157,6 +163,11 @@ def script_name() -> str:
                         help="Use nvidia docker",
                         action='store_true')
 
+    parser.add_argument("--shm-size",
+                        help="Size of the shared memory allocated for the container (e.g '1g')",
```

Comment:
Could you elaborate what this is used for? With whom is the memory being shared with?

Reply:
That's the size of the /dev/shm inside the container, which is used for interprocess communication for `DataLoader`s. By default it was only 50m, which causes hang ups when used for real-world example like in the tutorials, with 32 workers. https://docs.docker.com/engine/reference/commandline/run/